tools/upload-github.sh

```diff
@@ -0,0 +1,141 @@
+#!/bin/bash
+
+set -e
+
+# Generate GitHub release for a Godot version and upload artifacts.
+#
+# Usage: ./upload-github.sh -v 3.6
+# Usage: ./upload-github.sh -v 3.6 -f beta3
+# Usage: ./upload-github.sh -v 3.6 -f beta3 -r owner/repository
+#
+# Run this script from the root of the godot-build-scripts folder
+# after building Godot.
+
+# Folder this script is called from, a.k.a its working directory.
+export basedir=$(pwd)
+
+# Folder where this scripts resides in.
+scriptpath=$(readlink -f "$0")
+scriptdir=$(dirname "$scriptpath")
+# Root folder of this project, hopefully.
+export buildsdir=$(dirname "$scriptdir")
+
+if [ ! -d "${basedir}/releases" ] || [ ! -d "${basedir}/sha512sums" ] || [ ! -d "${basedir}/tmp" ]; then
```

Comment:
The `sha512sums` folder is mostly a local backup. We could just as well use `releases/**/SHA512-SUMS.txt` directly, no?

Reply:
Isn't the sha512sums folder the source of these files? I thought it was generated there first then copied to releases, but maybe I misread something. In any case, both sources are available indeed, I can refer to either, no hard preference.